pydicom/dataset.py

```diff
@@ -60,47 +60,51 @@ class PropertyError(Exception):
 
 
 class PrivateBlock(object):
-    """Helper class for a private block in the dataset.
-    (See PS3.5, Section 7.8.1 - Private Data Element Tags)
+    """Helper class for a private block in the :class:`Dataset`.
+
+    See the DICOM Standard, Part 5,
+    :dcm:`Section 7.8.1<part05/sect_7.8.html#sect_7.8.1>` - Private Data
+    Element Tags
 
     Attributes
     ----------
-    group : 32 bit int
-        The private group where the private block is located.
+    group : int
+        The private group where the private block is located as a 32-bit
+        :class:`int`.
     private_creator : str
         The private creator string related to the block.
     dataset : Dataset
         The parent dataset.
-    block_start : 32 bit int
-        The start element of the private block.
-        Note that the 2 low order hex digits of the element are always 0.
+    block_start : int
+        The start element of the private block as a 32-bit :class:`int`. Note
+        that the 2 low order hex digits of the element are always 0.
     """
 
     def __init__(self, key, dataset, private_creator_element):
         """Initializes an object corresponding to a private tag block.
 
         Parameters
         ----------
-        key : tuple (int, str)
+        key : tuple of int or tuple of str
```

Comment:
This is not correct - this is actually an (int, str) tuple.

Reply:
Whoops! You're right...